base/src/main/java/org/mozilla/jss/asn1/SEQUENCE.java

```diff
@@ -403,14 +403,13 @@ public ASN1Value decode(Tag tag, InputStream istream)
 
                     // skip over items that don't match.  Hopefully they are
                     // optional or have a default.  Otherwise, it's an error.
-                    Element e = elements.elementAt(index);
+                    Element e = elements.get(index);
                     if ((lookAhead == null) || lookAhead.isEOC() ||
                             !e.tagMatch(lookAhead.getTag())) {
                         if (e.isRepeatable()) {
                             repeatableElement = true;
                         } else if (e.isOptional()) {
                             // put an empty entry into the SEQUENCE
-                            SEQUENCE.Element se = new SEQUENCE.Element(null, null);
```

Comment:
SonarCloud is telling us that with the removal of `se` the private inner class `SEQUENCE.ELEMENT` is now unused, so you can also remove that.